.travis.yml

```diff
@@ -65,6 +65,9 @@ matrix:
         - emulator -avd test -no-skin -no-window &
       install:
         - yarn
+        - echo y | sdkmanager 'ndk-bundle'
```

Comment:
Can you add a note somewhere here to look at `android/ndk-samples` travis config https://github.com/android/ndk-samples/blob/master/.travis.yml#L16 in case something breakes?

Reply:
fixed in https://github.com/Nozbe/WatermelonDB/pull/693/commits/2cbf4e017607f44d99e8e8c064ba1f01eab9bce1